app/views/books/index.html.haml

```diff
@@ -1,20 +1,22 @@
 %h1.card-title= t('.books')
-
-.dropdown
-  %button#dropdownMenuButton1.btn.dropdown-toggle{"aria-expanded" => "false", "data-bs-toggle" => "dropdown", type: "button"}
-    = t('.categories')
-  %ul.dropdown-menu{ "aria-labelledby" => "dropdownMenuButton1" }
-    %li
-      = link_to t('.all'), root_path, class: "dropdown-item"
-    - @facade.categories.each do |category|
+- if @facade.books.blank?
```

Comment:
add a method to the facade where you will check if there are any books.

Also, please check differences between blank?, empty?, any?, present?, exists? methods.